src/arguments.rs

```diff
@@ -74,7 +76,11 @@ pub enum Command {
         cmd: Option<ConfigSubCommand>,
     },
 }
-
+#[derive(Debug, StructOpt)]
+pub enum EntryType {
+    Project,
+    Entry,
```

Comment:
and this to `TimeEntry` so it's consistent everywhere.
```suggestion
    TimeEntry,
```